addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -250,7 +273,7 @@ export class MessageList extends Component {
      */
     get orderedMessages() {
         const threadCache = this.threadView.threadCache;
-        if (this.props.order === 'desc') {
+        if (this.threadView.order === 'desc') {
```

Comment:
this whole method should be a simple compute on the model instead

Reply:
Witch model should I used ?
I have tried multiple model (`threadView`, `threadViewer`), but it doesn't work well. Most issue are with `threadCache` being `undefined` for some reason.

Reply:
On `threadView`, is `threadCache` a temporary issue? It's fine to add a condition for when it's empty inside a compute.